### PR TITLE
fixed actor match

### PR DESCRIPTION
**Pull request by killfishsquidswallow**

### CODE REVIEW

**Comment by DirtyRacer1337** [comment]
you tested it?

**Comment by killfishsquidswallow** [comment]
Tested with pornplus and tiny4k. There might be other things that need fixed

File: Contents/Code/networkPornPros.py [hunk 1]
```diff
@@ -79,8 +79,8 @@ def update(metadata, lang, siteNum, movieGenres, movieActors, art):
     metadata.collections.add(siteName)
 
     # Actor(s)
-    if 'pornplus' in sceneURL:
-        actors = detailsPageElements.xpath('//div[contains(@class, "space-y-4 p-4")]//a[contains(@href, "/models/")]')
+    if 'pornplus' or 'Tiny4k' in sceneURL:
+        actors = detailsPageElements.xpath('//div[contains(@class, "w-full flex flex-col space-y-4 p-4")]//a[contains(@href, "/models/")]')
```
Comment (DirtyRacer1337) [inline Contents/Code/networkPornPros.py:83]: `w-full flex flex-col` not needed

File: Contents/Code/networkPornPros.py [hunk 2]
```diff
@@ -79,8 +79,8 @@ def update(metadata, lang, siteNum, movieGenres, movieActors, art):
     metadata.collections.add(siteName)
 
     # Actor(s)
-    if 'pornplus' in sceneURL:
-        actors = detailsPageElements.xpath('//div[contains(@class, "space-y-4 p-4")]//a[contains(@href, "/models/")]')
+    if 'pornplus' or 'Tiny4k' in sceneURL:
```
Comment (DirtyRacer1337) [inline Contents/Code/networkPornPros.py:82]: I'm sure it's `tiny4k` in url

Comment (killfishsquidswallow) [reply Contents/Code/networkPornPros.py:82]: Thanks, updated

Comment (DirtyRacer1337) [reply Contents/Code/networkPornPros.py:82]: again, lowercase

**Pull request merged**